docs/source/tutorials/getting-started.ipynb

```diff
@@ -0,0 +1,675 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# Background\n",
+    "PSI/J (Portable Submission Interface for Jobs), is a Python abstraction layer which allows your HPC application to run virtually anywhere. <br> <br>\n",
+    "![psij-overview-visual](./images/psij-overview.png)\n",
```

Comment:
would propose to make this figure smaller (similar as for other figure in this doc is done)